packages/ihe-gateway/ihe-gateway-setup.xml

```diff
@@ -5256,1638 +3962,9 @@
-      <includeNewChannels>false</includeNewChannels>
-      <enabledChannelIds>
-        <string>660f9e44-4071-4842-8b5f-f2318bf89deb</string>
-        <string>64436c91-4c18-4f85-9d6f-e942b70e7fd2</string>
-        <string>2e17f9d4-fb39-43ba-87dd-72bac26789ef</string>
-        <string>f7efe8c5-8f5a-4312-9376-bee8cce106a1</string>
-      </enabledChannelIds>
-      <disabledChannelIds>
-        <string>2dfcf0ed-e0bf-4bff-b3bd-44b9b05b883f</string>
-        <string>3f38affa-7132-4c4d-8597-2679b2651a13</string>
-        <string>7a67af34-88c7-4477-9a33-25176d345ba4</string>
-        <string>cbd49d4a-24fa-40b6-aa12-d59d64168dd1</string>
-        <string>c7b1fb54-6dce-410c-a16a-e3ba6b6c6722</string>
-        <string>267b9cd3-a9cd-47e8-86ef-d1d3690707c7</string>
-        <string>f6899f98-6671-44f9-a764-d5f7c1858f02</string>
-        <string>2f3a1187-8950-4a03-bc93-181ff1ecb627</string>
-        <string>4ce23cec-c042-4608-b258-070cdad2d52d</string>
-        <string>7c527d59-374c-4bfe-ae96-863cd239d9d9</string>
-        <string>0223c1ab-b8ed-487d-82c2-341ae0a4eb9d</string>
-        <string>574e7271-1b21-4b79-82c3-fead249fc450</string>
-        <string>ce8aa1cf-5df6-4825-82c9-b51d5b2dfcfa</string>
-        <string>ed739638-6120-4aee-bcb2-acc661f2b12b</string>
-        <string>1a53f2e3-00e1-4de0-ba87-8bd096c00033</string>
-      </disabledChannelIds>
-      <codeTemplates>
-        <codeTemplate version="4.4.2">
-          <id>8ce40654-9675-40fa-ad52-770fad9f5509</id>
-          <name>Chache XSLT File</name>
-          <revision>1</revision>
-          <lastModified>
-            <time>1700976822789</time>
-            <timezone>GMT</timezone>
-          </lastModified>
-          <contextSet>
-            <delegate>
-              <contextType>DESTINATION_DISPATCHER</contextType>
-              <contextType>GLOBAL_PREPROCESSOR</contextType>
-              <contextType>CHANNEL_POSTPROCESSOR</contextType>
-              <contextType>GLOBAL_DEPLOY</contextType>
-              <contextType>SOURCE_FILTER_TRANSFORMER</contextType>
-              <contextType>CHANNEL_BATCH</contextType>
-              <contextType>CHANNEL_PREPROCESSOR</contextType>
-              <contextType>DESTINATION_FILTER_TRANSFORMER</contextType>
-              <contextType>GLOBAL_POSTPROCESSOR</contextType>
-              <contextType>GLOBAL_UNDEPLOY</contextType>
-              <contextType>CHANNEL_ATTACHMENT</contextType>
-              <contextType>CHANNEL_DEPLOY</contextType>
-              <contextType>SOURCE_RECEIVER</contextType>
-              <contextType>CHANNEL_UNDEPLOY</contextType>
-              <contextType>DESTINATION_RESPONSE_TRANSFORMER</contextType>
-            </delegate>
-          </contextSet>
-          <properties class="com.mirth.connect.model.codetemplates.BasicCodeTemplateProperties">
-            <type>FUNCTION</type>
-            <code>/**
-              Cache XSLT files in the global map
-
-              @param {String} path - full path to the file
```

Comment:
there's a lot of code templates being removed here - have you restored the latest code templates into Mirth before creating this backup?